### PR TITLE
NETOBSERV-1693 Prompt for copying files on exit followup

### PR DESCRIPTION
**Pull request by jpinsonneau**
## Description

@Amoghrd would that work for you ?

```
--copy=true|false|prompt
```

by default the `prompt` is asked at the end of the collection. If the user specify `--copy=true` or `--copy=false`, the corresponding action is done without prompt.

## Dependencies

n/a

## Checklist

If you are not familiar with our processes or don't know what to answer in the list below, let us know in a comment: the maintainers will take care of that.

* [X] Is this PR backed with a JIRA ticket? If so, make sure it is written as a title prefix _(in general, PRs affecting the NetObserv/Network Observability product should be backed with a JIRA ticket - especially if they bring user facing changes)._
* [ ] Does this PR require product documentation?
  * [ ] If so, make sure the JIRA epic is labelled with "documentation" and provides a description relevant for doc writers, such as use cases or scenarios. Any required step to activate or configure the feature should be documented there, such as new CRD knobs.
* [ ] Does this PR require a product release notes entry?
  * [ ] If so, fill in "Release Note Text" in the JIRA.
* [ ] Is there anything else the QE team should know before testing? E.g: configuration changes, environment setup, etc.
  * [ ] If so, make sure it is described in the JIRA ticket.
* QE requirements (check 1 from the list):
  * [X] Standard QE validation, with pre-merge tests unless stated otherwise.
  * [ ] Regression tests only (e.g. refactoring with no user-facing change).
  * [ ] No QE (e.g. trivial change with high reviewer's confidence, or per agreement with the QE team).

### CODE REVIEW

File: scripts/functions.sh [hunk 1]
```diff
@@ -267,6 +271,13 @@ function check_args_and_apply() {
         key="${option%%=*}"
         value="${option#*=}"
         case "$key" in
+            --copy) # Copy or skip without prompt
+                if [[ "$value" == "true" || "$value" == "false" || "$value" == "prompt" ]]; then
+                  copy="$value"
+                else
+                  echo "invalid value for --copy"
+                fi
+                ;;
```
Comment (msherif1234) [inline scripts/functions.sh:280]: pls update doc with the new option

**Comment by Amoghrd** [comment]
The `copy` option does not show up in help and also in env log lines. Otherwise works as expected!

File: scripts/functions.sh (hunk 1 above)
Comment (jpinsonneau) [reply scripts/functions.sh:288]: https://github.com/netobserv/network-observability-cli/pull/65/commits/c5139edefb4064ec20cf0eb6aec67293ef8a6944

**Comment by jpinsonneau** [comment]
@msherif1234 & @Amoghrd 

Rebased and added help, doc and an `echo` when param is set. Thanks for your feedback !

**Comment by Amoghrd** [comment]
Works as expected.
/label qe-approved

**Pull request merged**